### ISSUE TITLE
Install Storybook performance addon

### ISSUE THREAD

**Issue by elglup** [Royal-Navy/design-system#1229]
Install & test the storybook-addon-performance

**Comment by m7kvqbe1** [comment]
This ticket has been put on hold and thrown back into the pile:

- Untranspiled package and Storybook 6 (https://github.com/atlassian-labs/storybook-addon-performance/issues/53).

- The addon does not currently work with `styled-components`.

- There is not currently a stable release of the addon.

**Comment by m7kvqbe1** [comment]
Looks like this is good to go again (https://github.com/atlassian-labs/storybook-addon-performance/issues/53).

**Comment by hxltrhuxze** [comment]
Closing as this is a "nice-to-have" that has lingered for over a year now as we are still blocked by an external dependency. If this becomes a requirement again, it should resurface itself.
